types/canonical_json.go

```diff
@@ -3,14 +3,13 @@ package types
 import (
 	"time"
 
-	"github.com/tendermint/go-amino"
 	cmn "github.com/tendermint/tmlibs/common"
 )
 
 // Canonical json is amino's json for structs with fields in alphabetical order
 
 // TimeFormat is used for generating the sigs
-const TimeFormat = amino.RFC3339Millis
+const TimeFormat = "2006-01-02T15:04:05.000Z" // forced microseconds
```

Comment:
Can this lead to divergence? Would it make sense to change the `amino` format?

Reply:
Not related to the actual commits I've made (the proto files). But good question! 
This was recently changed in https://github.com/tendermint/go-amino/pull/175